open-metadata-resources/open-metadata-deployment/docker/egeria/Dockerfile

```diff
@@ -23,29 +23,29 @@ LABEL org.opencontainers.image.vendor = "ODPi" \
       org.opencontainers.image.version = "$version" \
       org.opencontainers.image.documentation = "https://egeria.odpi.org/open-metadata-resources/open-metadata-deployment/docker/egeria/" \
       org.opencontainers.image.ext.vcs-date = "$VCS_DATE" \
-      org.opencontainers.image.ext.docker.cmd = "docker run -d -p 8080:8080 odpi/egeria" \
-      org.opencontainers.image.ext.docker.cmd.devel = "docker run -d -p 8080:8080 -p 5005:5005 -e JAVA_DEBUG=true odpi/egeria" \
+      org.opencontainers.image.ext.docker.cmd = "docker run -d -p 9443:9443 odpi/egeria" \
```

Comment:
The ports are also documented on docker hub so I'll need to update there (easy just shouldn't forget)